packages/driver/patches/jquery+3.7.1.dev.patch

```diff
@@ -22,95 +22,120 @@ index 411a859..0000000
 -
--AMD is a module format built for the browser. For more information, we recommend [require.js' documentation](http://requirejs.org/docs/whyamd.html).
+-AMD is a module format built for the browser. For more information, we recommend [require.js' documentation](https://requirejs.org/docs/whyamd.html).
 -
 -```js
--define(["jquery"], function($) {
+-define( [ "jquery" ], function( $ ) {
 -
--});
+-} );
 -```
 -
 -### Node
 -
--To include jQuery in [Node](nodejs.org), first install with npm.
+-To include jQuery in [Node](https://nodejs.org/), first install with npm.
 -
 -```sh
 -npm install jquery
 -```
 -
--For jQuery to work in Node, a window with a document is required. Since no such window exists natively in Node, one can be mocked by tools such as [jsdom](https://github.com/tmpvar/jsdom). This can be useful for testing purposes.
+-For jQuery to work in Node, a window with a document is required. Since no such window exists natively in Node, one can be mocked by tools such as [jsdom](https://github.com/jsdom/jsdom). This can be useful for testing purposes.
 -
 -```js
--require("jsdom").env("", function(err, window) {
--	if (err) {
--		console.error(err);
--		return;
--	}
--
--	var $ = require("jquery")(window);
--});
+-const { JSDOM } = require( "jsdom" );
+-const { window } = new JSDOM( "" );
+-const $ = require( "jquery" )( window );
 -```
 diff --git a/node_modules/jquery/dist/jquery.js b/node_modules/jquery/dist/jquery.js
-index 773ad95..84ca2f6 100644
+index 1a86433..1c417b1 100644
 --- a/node_modules/jquery/dist/jquery.js
 +++ b/node_modules/jquery/dist/jquery.js
-@@ -1100,19 +1100,17 @@ setDocument = Sizzle.setDocument = function( node ) {
- 	docElem = document.documentElement;
- 	documentIsHTML = !isXML( document );
- 
--	// Support: IE 9-11, Edge
--	// Accessing iframe documents after unload throws "permission denied" errors (jQuery #13936)
--	if ( preferredDoc !== document &&
--		(subWindow = document.defaultView) && subWindow.top !== subWindow ) {
-+	// Support: IE 9 - 11+, Edge 12 - 18+
-+	// Accessing iframe documents after unload throws "permission denied" errors (see trac-13936)
-+	// Support: IE 11+, Edge 17 - 18+
-+	// IE/Edge sometimes throw a "Permission denied" error when strict-comparing
-+	// two documents; shallow comparisons work.
-+	// eslint-disable-next-line eqeqeq
-+	if ( docElem.msMatchesSelector && preferredDoc != document &&
-+		( subWindow = document.defaultView ) && subWindow.top !== subWindow ) {
- 
--		// Support: IE 11, Edge
--		if ( subWindow.addEventListener ) {
--			subWindow.addEventListener( "unload", unloadHandler, false );
--
--		// Support: IE 9 - 10 only
--		} else if ( subWindow.attachEvent ) {
--			subWindow.attachEvent( "onunload", unloadHandler );
--		}
-+		// Support: IE 9 - 11+, Edge 12 - 18+
-+		subWindow.addEventListener( "unload", unloadHandler );
- 	}
+@@ -1419,29 +1419,45 @@ find.matches = function( expr, elements ) {
+ };
+ 
+ find.matchesSelector = function( elem, expr ) {
```

Comment:
This change is essentially the same as [below](https://github.com/cypress-io/cypress/pull/30345/files#diff-af703d860398e63a491a39d6dc3ad85a888f4f8ac82a2e0d78435efd1b61c80eL36) except instead of setting `$.find.support.matchesSelector` to `false` to force `find` being called, we now just directly check `isUsingFocus` to force the same behavior.